Compilation error in cocotb/share/lib/vhpi/VhpiImpl.h

```
In file included from VhpiImpl.cpp:28:0:
VhpiImpl.h:192:9: error: ‘int VhpiSignalObjHdl::set_signal_value_binst(std::string&)’ marked override, but does not override
     int set_signal_value_binst(std::string &value) override;
         ^
VhpiImpl.cpp: In member function ‘GpiObjHdl* VhpiImpl::create_gpi_obj_from_handle(vhpiHandleT, std::string&, std::string&)’:
VhpiImpl.cpp:404:86: error: cannot allocate an object of abstract type ‘VhpiSignalObjHdl’
             new_obj = new VhpiSignalObjHdl(this, new_hdl, gpi_type, is_const(new_hdl));
                                                                                      ^
In file included from VhpiImpl.cpp:28:0:
VhpiImpl.h:173:7: note:   because the following virtual functions are pure within ‘VhpiSignalObjHdl’:
 class VhpiSignalObjHdl : public GpiSignalObjHdl {
       ^
In file included from VhpiImpl.h:31:0,
                 from VhpiImpl.cpp:28:
../gpi/gpi_priv.h:181:17: note: 	virtual int GpiSignalObjHdl::set_signal_value_binstr(std::string&)
     virtual int set_signal_value_binstr(std::string &value) = 0;
                     ^~~~~~~~~~~~~~~~~~~~~~~

```

gcc version 4.8.5 20150623 (Red Hat 4.8.5-36) (GCC)

@eric-wieser It seems related to https://github.com/cocotb/cocotb/pull/1149 ?

--- a/cocotb/share/lib/vhpi/VhpiImpl.h
+++ b/cocotb/share/lib/vhpi/VhpiImpl.h
@@ -189,7 +189,7 @@ public:
     using GpiSignalObjHdl::set_signal_value;
     int set_signal_value(long value) override;
     int set_signal_value(double value) override;
-    int set_signal_value_binst(std::string &value) override;
+    int set_signal_value_binstr(std::string &value) override;
     int set_signal_value_str(std::string &value) override;
 
     /* Value change callback accessor */

